src/App.jsx

```diff
@@ -1,9 +1,19 @@
-import { HomePage } from './pages';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { Header } from './components/organisms/Header/Header';
+import { HomePage, ProfilePage, EditProfilePage, ChatPage } from './pages';
 
 export const App = () => {
     return (
-        <main>
-            <HomePage />
-        </main>
+        <Router>
+            <Header/>
```

Comment:
We are missing the HomePage route (route with button to sign in) - this route shouldn't have header

And the current HomePage, should be FeedScreen (as it is on designs and miro)

Also we need default case for /404 page, if route is incorrect

Reply:
Done